components/toast/toast.css

```diff
@@ -0,0 +1,15 @@
+.toast {
+  align-items: center;
+  display: flex;
+  height: auto;
+  justify-content: space-between;
+  margin: auto;
+  max-width: 400px;
+  overflow-x: scroll;
```

Comment:
are you giving overflow to toast? Is it needed? Please clarify

Reply:
In case of extra long msg I want to make it scrollable. It can happen in some use-cases.